policy/modules/kernel/devices.fc

```diff
@@ -54,6 +54,7 @@
 /dev/hwrng		-c	gen_context(system_u:object_r:random_device_t,s0)
 /dev/gpiochip[0-9]+		-c	gen_context(system_u:object_r:gpio_device_t,s0)
 /dev/i915		-c	gen_context(system_u:object_r:dri_device_t,s0)
+/dev/iio:device[0-9]	-c	gen_context(system_u:object_r:iio_device_t,s0)
```

Comment:
The number suffix should rather be `[0-9]+` - according to the kernel code it could range up to `INT_MAX` in theory.

Reply:
Thank you, funny thing is that I put the + into the commit message.